src/main/java/com/zzpj/backend/controllers/AlcoholController.java

```diff
@@ -0,0 +1,51 @@
+package com.zzpj.backend.controllers;
+
+import com.zzpj.backend.entities.Alcohol;
+import com.zzpj.backend.services.AlcoholServiceLocal;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/alcohols")
+public class AlcoholController {
+
+    private AlcoholServiceLocal alcoholService;
+
+    @Autowired
+    public AlcoholController(AlcoholServiceLocal alcoholService) {
+        this.alcoholService = alcoholService;
+    }
+
+    @GetMapping
+    public List<Alcohol> getAll(){
+        return alcoholService.getAllAlcohols();
+    }
+
+    @GetMapping("{id}")
+    public Alcohol get(@PathVariable Long id){
+        return alcoholService.getAlcohol(id);
+    }
+
+    @PostMapping
+    public String add(@RequestBody Alcohol alcohol){
```

Comment:
We should add here validation:
`@Validated @ModelAttribute("alcohol") `
And add appropriate validation in our model class
` @NotBlank(message = "")`